lib/proxy/router.go

```diff
@@ -241,6 +241,9 @@ func (r *Router) DialHost(ctx context.Context, clientSrcAddr, clientDstAddr net.
 		serverAddr      string
 		proxyIDs        []string
 		sshSigner       ssh.Signer
+
+		// DELETE IN 15.0.0
```

Comment:
```suggestion
		// DELETE IN 15.0.0 (capnspacehook)
```

So that I can chase you down :-)

Reply:
@capnspacehook  `master` branch basically _is_ 15.0.0. Can we just not include these "DELETE in 15" things in this PR at all and only include them in the branch/v14 backport?

Reply:
I was wondering about that actually, that seems like the best course of action. Changed so this will merge directly into branch/v14.